### PR TITLE
Improve package manager detection

### PR DESCRIPTION
**Pull request by dev-afzalansari**
closes: #64 

When no package manager is used yet in a fresh remix project the cli will now prompt an extra question for the name of the package manager only if the user has chosen to install the dependencies after creating the remix-pwa project. if the user has chosen to not install the deps it won't prompt for that question.

As expected if the cli can detect the package manager it will prompt a question for that package manager.

### CODE REVIEW

**Comment by dev-afzalansari** [comment]
Sorry,
I tried to target `dev` branch but my commits were on `main` branch

**Comment by dev-afzalansari** [comment]
@ShafSpecs,
Is it okay if I target the `main` branch since it's the branch for the latest release?

**Comment by ShafSpecs** [comment]
> @ShafSpecs,
> Is it okay if I target the `main` branch since it's the branch for the latest release?

@dev-afzalansari Yes, you can.

**Comment by dev-afzalansari** [comment]
@ShafSpecs,
Now it's ready for review.

**Comment by ShafSpecs** [comment]
> @ShafSpecs,
> Now it's ready for review.

Would do that first thing later today when I load up my PC 👍

File: cli.ts [hunk 1]
```diff
@@ -1,19 +1,18 @@
 #!/usr/bin/env node
 
-const fse = require("fs-extra");
+import * as fse from "fs-extra";
```
Comment (ShafSpecs) [inline cli.ts:3]: This is not an esm module, `import`s don't work. And let's stick with `require` to keep a uniform import syntax

**Review by ShafSpecs — CHANGES_REQUESTED** [review]
Looks good 👍, why did you switch to import though for `fs-extra`?

File: cli.ts (hunk 1 above)
Comment (dev-afzalansari) [reply cli.ts:3]: yep, sounds good. I will do this as first thing tomorrow(it's night here).

Comment (ShafSpecs) [reply cli.ts:3]: Alright 👍

**Review by ShafSpecs — APPROVED** [review]
Good to go 👍

**Pull request merged**